guide/source/program_administrator/identify_data_needs.rst

```diff
@@ -301,16 +295,17 @@ immediately.  Follow the steps below to submit additional requests if needed:
    `HPXML GitHub issues page <https://github.com/hpxmlwg/hpxml/issues/>`_.
 #. Click "New Issue"
 #. Fill out the form to ask your question or make your request. No need to assign a person or milestone.
-#. Check the appropriate labels: probably "docs" and "enhancement" or "bug".
+#. Check the appropriate labels: probably "docs" and "enhancement" or "bug". [THESE THREE CATEGORIES SHOULD PROBABLY BE DEFINED SOMEWHERE. THEY ARE NOT EXPLICITLY OBVIOUS AS TO WHEN TO USE WHICH.
```

Comment:
This label issue isn't that important. It's just a way of organizing issues. I can apply the appropriate labels as they come in. Let's nix this line.